lib/assessment.js

```diff
@@ -39,7 +71,15 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
+  if (!obj) {
+    var target = {};
+  } else {
+    var target = obj;
+  }
+
+  target.prototype.propertyName = method;
 
+  return target;
```

Comment:
Otherwise, your solution is basically right. A shorter version, however, might be: 
```javascript
obj = obj || {};
obj[propertyName] = method;
return obj;
```